pkg/install/installer.go

```diff
@@ -315,43 +304,148 @@ func (i *Installer) InstallCMakeIntegration() error {
-		// /(lib/|lib|share)/<name>*/(cmake|CMake)/
-		// See:
-		// https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure
-		// https://gitlab.kitware.com/cmake/cmake/-/blob/5ed9232d781ccfa3a9fae709e12999c6649aca2f/Modules/Platform/UnixPaths.cmake#L30)
-		_, err := i.copyCMakeIntegration("/usr/local/share/cifuzz")
-		if err != nil {
-			return err
-		}
+	cmakeSrc := filepath.Join(i.projectDir, "tools", "cmake", "cifuzz")
+	destDir := i.shareDir()
+	opts := copy.Options{
+		// Skip copying the replayer, which is a symlink on UNIX but checked out
+		// by git as a file containing the relative path on Windows. It is
+		// handled below.
+		OnSymlink: func(string) copy.SymlinkAction {
+			return copy.Skip
+		},
 	}
-	dirForRegistry, err := i.copyCMakeIntegration(i.shareDir())
+	err = copy.Copy(cmakeSrc, destDir, opts)
 	if err != nil {
-		return err
+		return errors.WithStack(err)
 	}
-	return registerCMakePackage(dirForRegistry)
+
+	// Copy the replayer, which is a symlink and thus may not have been copied
+	// correctly on Windows.
+	replayerSrc := filepath.Join(i.projectDir, "tools", "replayer", "src", "replayer.c")
+	replayerDir := filepath.Join(destDir, "src")
+	err = os.MkdirAll(replayerDir, 0755)
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	err = copy.Copy(replayerSrc, filepath.Join(replayerDir, "replayer.c"))
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	err = copy.Copy(replayerSrc, filepath.Join(replayerDir, "replayer.cpp"))
+	if err != nil {
+		return errors.WithStack(err)
+	}
+
+	return nil
+}
+
+func CIFuzzExecutablePath(binDir string) string {
+	path := filepath.Join(binDir, "cifuzz")
+	if runtime.GOOS == "windows" {
+		path += ".exe"
+	}
+	return path
 }
 
-func (i *Installer) PrintPathInstructions() {
+func PrintPathInstructions(binDir string) {
 	if runtime.GOOS == "windows" {
 		// TODO: On Windows, users generally don't expect having to fiddle with their PATH. We should update it for
 		//       them, but that requires asking for admin access.
-		fmt.Fprintf(os.Stderr, `
-Please add the following directory to your PATH:
+		_, _ = fmt.Fprintf(os.Stderr, `Please add the following directory to your PATH:
     %s
-`, i.binDir())
+`, binDir)
 	} else {
-		fmt.Fprintf(os.Stderr, `
-Please add the following to ~/.profile or ~/.bash_profile:
+		_, _ = fmt.Fprintf(os.Stderr, `Please add the following to ~/.profile or ~/.bash_profile:
     export PATH="$PATH:%s"
-`, i.binDir())
+`, binDir)
 	}
 }
 
-func findProjectDir() (string, error) {
+// ExtractBundle extracts all installation files from bundle into targetDir and registers the CMake package
+func ExtractBundle(targetDir string, bundle *embed.FS) error {
+	// List of files which have to be made executable
+	executableFiles := []string{
+		"bin/cifuzz",
+		"bin/minijail0",
+		"lib/libminijailpreload.so",
+		"lib/process_wrapper",
+	}
+
+	targetDir, err := validateTargetDir(targetDir)
+	if err != nil {
+		return errors.WithStack(err)
```

Comment:
This attaches a duplicate stack trace, we already attach one in `validateTargetDir`.
```suggestion
		return err
```